internals/link_helpers.py

```diff
@@ -0,0 +1,87 @@
+import re
+import requests
+import json
+import logging
+from typing import Any
+
+LINK_TYPE_CHROMIUM_BUG = 'chromium_bug'
+LINK_TYPE_UNKNOWN = 'unknown'
+LINK_TYPES_REGEX = {
+    # https://bugs.chromium.org/p/chromium/issues/detail?id=
+    LINK_TYPE_CHROMIUM_BUG: re.compile(r'https://bugs\.chromium\.org/p/chromium/issues/detail\?.*'),
+    # any other links
+    LINK_TYPE_UNKNOWN: re.compile(r'https?://.*'),
+}
+
+
+class Link():
```

Comment:
It's potentially confusing to have `class Link` defined in a file that is not `link.py`.
Maybe you should rename `link.py` to `feature_links.py`.

Also, I am not sure that you need class `Link`.    I think you could make the methods of `Link` into top-level functions in this file and have them return tuples rather than constructing a `Link` object.  That might be easier to understand than using the current names `FeatureLinks` and `Link`, which are logically similar to each other.